tensorbay/opendataset/AADB/loader.py

```diff
@@ -22,7 +22,7 @@
 
 
 def AADB(path: str) -> Dataset:
-    """Load the AADB to TensorBay.
+    """`AADB <https://www.ics.uci.edu/~skong2/aesthetics.html>`_ dataset.
```

Comment:
about the commit message:
1. `A of B` is better than `B's A`
2. `citations` is ambiguous, will `link` be better?

Reply:
agree!